hirundo/dataset_optimization.py

```diff
@@ -597,3 +795,31 @@ def cancel(self) -> None:
         if not self.run_id:
             raise ValueError("No run has been started")
         self.cancel_by_id(self.run_id)
+
+
+class DataOptimizationDatasetOut(BaseModel):
+    id: int
+
+    name: str
+    labeling_type: LabelingType
+
+    storage_integration: ResponseStorageIntegration
+
+    data_root_url: HirundoUrl
+
+    classes: typing.Optional[list[str]] = None
```

Comment:
If we do change `classes` to be a field of `Metadata` and not of `OprimizationDataset` then remember to change this as well :)

Reply:
I will keep that in mind in future. I am not planning on doing it yet